docker-compose.yml

```diff
@@ -7,8 +7,8 @@ services:
     image: postgres:14.4
     env_file: .env
     environment:
-      POSTGRES_DB: ${DB_NAME}
-      POSTGRES_PASSWORD: ${DB_PASSWORD}
+      POSTGRES_DB: $${DB_NAME}
+      POSTGRES_PASSWORD: $${DB_PASSWORD}
```

Comment:
was getting warning about not using $$

Reply:
It seems like this is causing my `password authentication failed` error, changing back to single dollar sign fixes for me.